DQM/PixelLumi/plugins/PixelLumiDQM.cc

```diff
@@ -430,7 +429,6 @@ void PixelLumiDQM::endLuminosityBlock(edm::LuminosityBlock const &lumiBlock, edm
                     (*it).second.dnumFP.at(1) + (*it).second.dnumFM.at(0) + (*it).second.dnumFM.at(1);
     if (useInnerBarrelLayer)
       etotal = (*it).second.dnumB.at(0);
```

Comment:
```suggestion
    double etotal = useInnerBarrelLayer 
                              ? (*it).second.dnumB.at(0)
                              : (*it).second.dnumB.at(1) + (*it).second.dnumB.at(2) + (*it).second.dnumFP.at(0) +
                                (*it).second.dnumFP.at(1) + (*it).second.dnumFM.at(0) + (*it).second.dnumFM.at(1);
```
can avoid some useless computation if `useInnerBarrelLayer`